### PR TITLE
Add old API endpoints back for now

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

File: webserver/views/api/test/test_legacy.py [hunk 1]
```diff
@@ -19,3 +20,20 @@ def test_submit_low_level(self):
         # Getting from the new API
         resp = self.client.get("/api/v1/%s/low-level" % mbid)
         self.assertEqual(resp.status_code, 200)
+
+    def test_get_low_level(self):
```
Comment (alastair) [inline webserver/views/api/test/test_legacy.py]: did we have tests for bad uuids?

Comment (gentlecat) [inline webserver/views/api/test/test_legacy.py]: There's a `_validate_data_arguments` function. Nothing has changed there. I just added old code back.

Comment (alastair) [inline webserver/views/api/test/test_legacy.py]: :+1:

**Comment by alastair** [comment]
looks good

**Pull request merged**